fintech-examples/fintech-ui/src/app/dashboard/show-accounts/show-accounts.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { AccountDetails } from '../../api';
+import { Subscription } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { AisService } from '../services/ais.service';
+
+@Component({
+  selector: 'app-show-accounts',
+  templateUrl: './show-accounts.component.html',
+  styleUrls: ['./show-accounts.component.scss']
+})
+export class ShowAccountsComponent implements OnInit, OnDestroy {
```

Comment:
Maybe a better naming would be: AccountListComponent, but its up to you

Reply:
done